.github/workflows/ci.yaml

```diff
@@ -30,11 +33,14 @@ jobs:
       - name: Git Diff
         run: git diff --exit-code
       - name: Go Build
-        run: CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -mod vendor -o manager main.go
+        run: CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -mod vendor -o manager main.go
   tests:
     name: Tests
     runs-on: ubuntu-latest
     container: docker.io/library/golang:latest
+    strategy:
+      matrix:
+        platform: ['amd64', 's390x']
```

Comment:
@modassarrana89 if I am not wrong the tests are running only on amd64. You may need to setup cross-arch builds.